catalog/core/catalog-core-standardframework/src/test/java/ddf/catalog/impl/operations/ResourceOperationsOptionsTest.java

```diff
@@ -0,0 +1,365 @@
+ * License, or any later version.
+ * <p/>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+
+package ddf.catalog.impl.operations;
+
+import static org.mockito.Matchers.any;
+import static org.mockito.Matchers.anyBoolean;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import java.io.Serializable;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import ddf.catalog.cache.solr.impl.ValidationQueryFactory;
+import ddf.catalog.content.impl.MockMemoryStorageProvider;
+import ddf.catalog.data.Result;
+import ddf.catalog.data.impl.MetacardImpl;
+import ddf.catalog.federation.FederationException;
+import ddf.catalog.federation.FederationStrategy;
+import ddf.catalog.filter.proxy.adapter.GeotoolsFilterAdapterImpl;
+import ddf.catalog.filter.proxy.builder.GeotoolsFilterBuilder;
+import ddf.catalog.impl.FrameworkProperties;
+import ddf.catalog.impl.QueryResponsePostProcessor;
+import ddf.catalog.operation.QueryRequest;
+import ddf.catalog.operation.QueryResponse;
+import ddf.catalog.operation.ResourceRequest;
+import ddf.catalog.plugin.PostIngestPlugin;
+import ddf.catalog.plugin.PostResourcePlugin;
+import ddf.catalog.resource.ResourceNotFoundException;
+import ddf.catalog.source.CatalogProvider;
+import ddf.catalog.source.UnsupportedQueryException;
+import ddf.catalog.util.impl.SourcePoller;
+import ddf.mime.MimeTypeResolver;
+import ddf.mime.MimeTypeToTransformerMapper;
+import ddf.mime.mapper.MimeTypeMapperImpl;
+
+public class ResourceOperationsOptionsTest {
+    ResourceOperations resourceOperations;
+
+    FrameworkProperties frameworkProperties;
+
+    OperationsSecuritySupport opsSecuritySupport;
+
+    SourcePoller mockPoller;
+
+    CatalogProvider provider;
+
+    FederationStrategy mockFederationStrategy;
+
+    ArrayList<PostIngestPlugin> postIngestPlugins;
+
+    MockMemoryStorageProvider storageProvider;
+
+    MimeTypeResolver mimeTypeResolver;
+
+    MimeTypeToTransformerMapper mimeTypeToTransformerMapper;
+
+    List<PostResourcePlugin> mockPostResourcePlugins;
+
+    ResourceRequest resourceRequestMock;
+
+    QueryResponse queryResponseMock;
+
+    QueryOperations queryOperationsMock;
+
+    boolean fanoutEnabled;
+
+    URI testUri;
+
+    Map<String, Serializable> requestPropertiesMock;
+
+    MetacardImpl metacard;
+
+    QueryRequest queryRequestMock;
+
```

Comment:
Access modifiers *could* be important in test code, we don't want to forsake all best practices just because they are tests. (Can't speak for this situation specifically)